### DISCUSSION TITLE
Can't Seem to Configure Email Notifications for Outlook

### DISCUSSION THREAD

**Discussion by Inshakoor** [containrrr/watchtower#637] [Q&A] [answered]
Good morning!  I recently setup a new docker container in Portainer of Watchtower.  I remember struggling with my last iteration trying to get email notifications to work with my outlook.com email, but I did get it.  Of course, I didn't save the settings I had that got it to work and now I can't for the LIFE of me reconfigure email notifications. I've scoured the web and the official docs and just can't get it to work.  Please see the attached screenshot for the settings I'm using.  I get the following error: "Failed to send notification email: 504 5.7.4 Unrecognized authentication type [BY5PR13CA0005.namprd13.prod.outlook.com]".  I've tried multiple iterations of the settings with removing TLS or changing it to STARTTLS and bypassing TLS with WATCHTOWER_NOTIFICATION_EMAIL_SERVER_TLS_SKIP_VERIFY but I can't bloody remember how I got it to work and can't seem to find anything out there that's pointing me in the right direction.  I'd appreciate any suggestions that anyone might have.  Thank you for your time!

![Portainer_WatchTower_Vars](https://user-images.githubusercontent.com/57557992/92329510-ad920580-f01c-11ea-9252-cf8d921e2071.png)

**Comment by zoispag** [comment]
Thanks for moving the discussion here @Inshakoor .
@piksel could you help here?

**Comment by piksel** [comment, marked as answer]
This unfortunately due to [office365 not supporting PLAIN auth](https://github.com/containrrr/shoutrrr/issues/34).
I did manage to get XOAUTH2 working for Shoutrrr SMTP service, [but it's not complete](https://github.com/containrrr/shoutrrr/pull/44) yet (and not released).

[Official statement from Microsoft](https://support.microsoft.com/en-us/office/outlook-com-no-longer-supports-auth-plain-authentication-07f7d5e9-1697-465f-84d2-4513d4ff0145?ui=en-us&rs=en-us&ad=us)

**Reply by Inshakoor** [reply]
Sounds good.  I'll switch to gmail for now.  Thank you!
